app/config.js

```diff
@@ -49,3 +53,66 @@ export function getCfgPath() {
   const config = new Config();
   return config.path;
 }
+
+export function appDataDirectory() {
+  const path = require('path');
+  const os = require('os');
+
+  if (os.platform() == 'win32') {
+    return path.join(os.homedir(), 'AppData', 'Local', 'Decrediton');
+  } else if (process.platform === 'darwin') {
+    return path.join(os.homedir(), 'Library','Application Support','decrediton');
+  } else {
+    return path.join(os.homedir(),'.config','decrediton');
+  }
+}
+
+export function getCert() {
+  var cert = '';
+  var cfg = getCfg();
+  if (cfg.cert_path != '') {
+    return(cfg.cert_path);
+  }
+  var certPath = '';
+  if (os.platform() == 'win32') {
+    certPath = path.join(os.homedir(), 'AppData', 'Local', 'Decrediton', 'rpc.cert');
```

Comment:
That is what I originally had and it (and process.env.APPDATA) were both undefined when I ran it in the electron app (but had the value I expected with standalone node).